### PR TITLE
Feature/new new conversation system message cell zios 10271

### PR DESCRIPTION
**Pull request by daehn**
## What's in this PR?

* Add new model schema version including `allTeamUsersAdded` and `numberOfGuestsAdded` fields which will be used for the updated `.newConversation` system message.
* Add test for `.newConversation` system message when inserting new conversation.

### This PR depends on: 

- [x] The new model schema version is still missing a migration test.

### CODE REVIEW

File: Source/Model/Message/ZMMessage.m [hunk 1]
```diff
@@ -856,6 +856,8 @@ + (instancetype)createOrUpdateMessageFromUpdateEvent:(ZMUpdateEvent *)updateEven
     message.users = usersSet;
     message.text = messageText != nil ? messageText : name;
     
+    [message updateNewConversationPropertiesWithUsers:usersSet context:moc conversation:conversation];
```
Comment (typfel) [inline Source/Model/Message/ZMMessage.m]: Only necessary to run when `systemMessageType` is `ZMSystemMessageTypeNewConversation ?

Comment (daehn) [reply Source/Model/Message/ZMMessage.m]: Yeah, misleading naming, should be referring to new conversation system messages so maybe better to include that in the name.

Comment (daehn) [reply Source/Model/Message/ZMMessage.m]: @typfel changed the name slightly now.

**Comment by daehn** [comment]
@vytis updated!

**Comment by vytis** [comment]
@daehn ready to merge? Tests are green.

**Comment by daehn** [comment]
@vytis merged!

**Pull request merged**